lib/semantics/mod-file.cc

```diff
@@ -36,14 +36,15 @@ using namespace parser::literals;
 // are using another encoding.
 static constexpr auto magic{"\xef\xbb\xbf!mod$ v1 sum:"};
 
-static const SourceName *GetSubmoduleParent(const parser::Program &);
+static const std::optional<SourceName> GetSubmoduleParent(
```

Comment:
`const` doesn't add anything here.

Reply:
Thanks, I have removed the leftover `const`